frontend/src/components/CreateMemo/components/LocationMapPicker/Map.tsx

```diff
@@ -33,40 +31,39 @@ const MapForm = ({ selectedLocation, onMapClick }: any) => {
     const map = useMapEvents({
       click(e) {
         const clickedLocation = [e.latlng.lat, e.latlng.lng];
-        setPositions([...positions, clickedLocation]); // Add new position to the array
+        setPosition(clickedLocation); // Add new position to the array
         onMapClick(clickedLocation);
         setInitialPosition(clickedLocation);
       },
     });
 
     const deleteMarker = (index: any, e: any) => {
       e.stopPropagation(); // Stop event propagation to prevent the map click event
-      const updatedPositions = [...positions];
-      updatedPositions.splice(index, 1); // Remove the position at the specified index
-      setPositions(updatedPositions);
+      setPosition(null);
```

Comment:
The user shouldnt be allowed to delete the marker if there is only one marker to place. 

A location marker should me mandatory for a memo.

Reply:
removed the delete button and popup for the marker